dnadoh/synthesize.py

```diff
@@ -163,14 +170,29 @@ class PersonGenerator:
     WEIGHT_MEANS = {"F": 80.0, "M": 87.0, "O": 84.0}
     WEIGHT_RSD = 0.12
 
-    def __init__(self):
+    # Avg household size, per 2021 stats
+    # https://www.statista.com/statistics/242189/disitribution-of-households-in-the-us-by-household-size/
+    # key = household size, val = proportion of households with number of people indicated by key
+    HOUSEHOLD_SIZE_DISTRIB = {1: 0.2845,
+                              2: 0.3503,
+                              3: 0.1503,
+                              4: 0.1239,
+                              5: 0.0583,
+                              6: 0.0203,
+                              7: 0.0124}
+    UPPER_BOUND = 10 # Largest possible household size, according to our model
```

Comment:
would HH sizes of 8, 9, and 10 ever be selected since they're not in the dictionary above?